### PR TITLE
IOTEVENT-435: update to support zipkin reporter

### PR DESCRIPTION
**Pull request by edwardsoo-ss**
@jeff-blaisdell @Bijnagte take a look please?

### CODE REVIEW

File: st-version.txt [hunk 1]
```diff
@@ -1 +1 @@
-0.0.2-SNAPSHOT
\ No newline at end of file
+1.0.0-SNAPSHOT
```
Comment (edwardsoo-ss) [inline st-version.txt]: moving to 1.0.0 because collector support is dropped

**Comment by Bijnagte** [comment]
I would think we could just take the commits from the forked library.
https://github.com/hyleung/ratpack-zipkin/commit/891bfcfe3e476a0a320e8256df1a58201ea764b2 seems to do much of the same work.
We would like to eventually switch to using the upstream library and just contribute to that. This fork was just to deal with the lack of request interceptors in the existing ratpack httpclient.

File: src/main/java/ratpack/zipkin/ServerTracingModule.java [hunk 2]
```diff
@@ -115,7 +117,7 @@ public LocalTracer localTracer(final Brave brave) {
   
   public static class Config {
     private String serviceName = "unknown";
-    private SpanCollector spanCollector;
+    private Reporter<Span> spanReporter;
```
Comment (jeff-blaisdell) [inline src/main/java/ratpack/zipkin/ServerTracingModule.java:127]: Is upstream missing these changes?  It'd be nice to make a pull request there as well.

**Comment by edwardsoo-ss** [comment]
updated to just merging commits from forked lib to smartthings-master, removing major version change because merge has deprecated support for collector

**Comment by edwardsoo-ss** [comment]
closing in favor of rebasing `SmartThingsOSS:smartthings-master` on `hyleung:master`

**Pull request closed without merging**